### PR TITLE
chore(rest-plugin): modify header for rebuild history cli

### PR DESCRIPTION
**Pull request by abhilashshetty04**

### CODE REVIEW

**Review by tiagolobocastro — COMMENTED** [review]
Missing translation to bytes

File: control-plane/plugin/src/resources/utils.rs [hunk 1]
```diff
@@ -52,10 +52,10 @@ lazy_static! {
...
         "SRC",
         "STATE",
-        "BLKS-TOTAL",
-        "BLKS-RECOVERED",
-        "BLKS-TRANSFERRED",
-        "BLKS-SIZE",
+        "TOTAL",
+        "RECOVERED",
+        "TRANSFERRED",
+        "SIZE",
```
Comment (tiagolobocastro) [inline control-plane/plugin/src/resources/utils.rs]:
```suggestion
```

File: control-plane/plugin/src/resources/volume.rs [hunk 2]
```diff
@@ -268,10 +268,10 @@ impl CreateRows for openapi::models::RebuildHistory {
...
                     child_uuid(rec.src_uri.as_str()),
                     rec.rebuild_job_state,
-                    to_human_readable(rec.blocks_total),
-                    to_human_readable(rec.blocks_recovered),
-                    to_human_readable(rec.blocks_transferred),
-                    to_human_readable(rec.block_size),
+                    to_human_readable(rec.blocks_total * rec.block_size),
+                    to_human_readable(rec.blocks_recovered * rec.block_size),
+                    to_human_readable(rec.blocks_transferred * rec.block_size),
+                    to_human_readable(rec.block_size * rec.block_size),
```
Comment (tiagolobocastro) [inline control-plane/plugin/src/resources/volume.rs]:
```suggestion
```

**Review by tiagolobocastro — COMMENTED** [review]
Let's remove block size since it's not relevant here

File: control-plane/plugin/src/resources/volume.rs (hunk 2 above)
Comment (abhilashshetty04) [reply control-plane/plugin/src/resources/volume.rs]: Removed

**Review by tiagolobocastro — APPROVED** [review]

**Comment by abhilashshetty04** [comment]
bors merge

**Comment by abhilashshetty04** [comment]
bors try

**Comment by abhilashshetty04** [comment]
bors merge

**Pull request merged**